### PR TITLE
Adding a RISC0 version of `sdc`

### PR DESCRIPTION
**Pull request by Robertorosmaninho**
This PR introduces the zk_sdc using the RISC0 backend.
It still doesn't have input support as we didn't define one.

### CODE REVIEW

File: sdc/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#![deny(warnings)]
+use sdc::sdc;
+
+pub fn main() {
```
Comment (yzhang90) [inline sdc/src/main.rs:4]: add a comment saying this file is just a place holder.

**Review by yzhang90 — APPROVED** [review]
LGTM. The only concern is the folder structure. The sdc and sdc_risc0 are both at the top level. @xc93

**Comment by yzhang90** [comment]
please try it on larger input.

File: sdc/src/main.rs (hunk 1 above)
Comment (Robertorosmaninho) [reply sdc/src/main.rs:4]: Done!

**Pull request merged**